include/sys/aggsum.h

```diff
@@ -27,12 +27,13 @@ extern "C" {
 
 #define	CACHE_LINE_SIZE	64
```

Comment:
Should be removed.

Reply:
... I could have sworn I removed that. I must have exited without saving. Fixed.